src/legacy/core_plugins/kibana/public/management/sections/index_patterns/index_pattern_table/index_pattern_table.tsx

```diff
@@ -38,13 +39,15 @@ const columns = [
   {
     field: 'title',
     name: 'Pattern',
-    render: (name: string, { id }: { id: string }) => (
-      <EuiButtonEmpty size="xs" href={`#/management/kibana/index_patterns/${id}`}>
+    render: (name: string, props: { id: string; default: boolean }) => (
```

Comment:
Minor nit: I'd suggest using `item` or `indexPattern` instead of `props`, since I think that's more representative of what's being provided to this parameter. Plus, `props` immediately makes me think of React props, which is a little confusing in this context. 😄

Reply:
Definitly! thx!

Reply:
done